Apps/W1/HybridGP/app/src/pages/GPMigrationGeneralSettings.Page.al

```diff
@@ -0,0 +1,57 @@
+page 4102 "GP Migration General Settings"
+{
+    Caption = 'GP Migration General Settings';
+    PageType = Card;
+    ApplicationArea = All;
+    UsageCategory = Administration;
+
+    layout
+    {
+        area(Content)
+        {
+            label(DescriptionHeader)
+            {
+                ApplicationArea = All;
+                Caption = 'Description';
+                Style = Strong;
+            }
+            label(Intro)
+            {
+                ApplicationArea = All;
+                Caption = 'Use this page to configure general settings for the migration.';
+            }
+
+            group(OverallSettings)
+            {
+                Caption = 'Overall Settings';
+
+                field("TwoStepProcess"; UseTwoStepProcess)
+                {
+                    Caption = 'Use two step process';
+                    ToolTip = 'Specify if you want the migration to use the older 2 step process.';
```

Comment:
Might want different wording here. Those that have never used the tool will have no idea what this means. I don't have a good alternative for you at this time.

Reply:
I agree, I couldn't think of anything better either. This won't be the final version of the feature, I've asked Kasey and Iryna to help with wording and explanation text. Thanks for the feedback.